JavaScript (eslint)/eslint-config-hzdg-react/package.json

```diff
@@ -24,19 +24,19 @@
   },
   "homepage": "https://github.com/hzdg/linter-configs#readme",
   "devDependencies": {
-    "@babel/core": "7.0.0-beta.54",
-    "@babel/preset-env": "7.0.0-beta.54",
-    "@babel/register": "7.0.0-beta.54",
-    "ava": "^0.22.0",
-    "babel-eslint": "9.0.0-beta.3",
-    "eslint": "5.2.0",
+    "@babel/core": "7.1.6",
+    "@babel/preset-env": "7.1.6",
+    "@babel/register": "7.0.0",
+    "ava": "^0.25.0",
+    "babel-eslint": "10.0.1",
+    "eslint": "5.9.0",
     "eslint-config-hzdg": "0.9.0",
-    "eslint-config-prettier": "2.9.0",
-    "eslint-plugin-babel": "5.1.0",
-    "eslint-plugin-jsx-a11y": "6.1.1",
-    "eslint-plugin-prettier": "2.6.2",
-    "eslint-plugin-react": "7.10.0",
-    "prettier": "1.14.0"
+    "eslint-config-prettier": "3.3.0",
+    "eslint-plugin-babel": "5.3.0",
+    "eslint-plugin-jsx-a11y": "6.1.2",
+    "eslint-plugin-prettier": "3.0.0",
+    "eslint-plugin-react": "7.11.1",
+    "prettier": "1.15.2"
   },
   "peerDependencies": {
     "eslint": "5.1.0",
```

Comment:
We should update the peerDeps to match the versions of eslint and plugins that we're testing against.

Reply:
I've updated peer dependencies